dockstore-client/src/main/java/io/dockstore/client/cli/YamlClient.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *    Copyright 2022 OICR and UCSC
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.dockstore.client.cli;
+
+import java.util.List;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.Parameters;
+import io.dockstore.client.cli.YamlVerify.ValidateYamlException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.dockstore.client.cli.ArgumentUtility.out;
+import static io.dockstore.client.cli.JCommanderUtility.printJCommanderHelp;
+import static io.dockstore.client.cli.JCommanderUtility.printJCommanderHelpYaml;
+
+/**
+ *
+ */
+public final class YamlClient {
+
+    private static final Logger LOG = LoggerFactory.getLogger(YamlClient.class);
+
+
+    private YamlClient() {
+        // disable constructor for utility class
+    }
+
+
+    /**
+     * @param args
+     */
+    public static boolean handleCommand(List<String> args) {
+        String[] argv = args.toArray(new String[args.size()]);
+        JCommander jc = new JCommander();
+
+        CommandYaml commandYaml = new CommandYaml();
+        JCommander jcPlugin = JCommanderUtility.addCommand(jc, "yaml", commandYaml);
+        CommandYamlValidate commandYamlValidate = new CommandYamlValidate();
+        commandYamlValidate = new CommandYamlValidate();
+        JCommanderUtility.addCommand(jcPlugin, "validate", commandYamlValidate);
+
+        // Not parsing with jc because we know the first command was plugin.  jc's purpose is to display help
+        jcPlugin.parse(argv);
+        try {
+            if (args.isEmpty() || commandYaml.help) {
+                printJCommanderHelpYaml(jc, "dockstore", "yaml");
```

Comment:
"yaml" gets used a lot in this whole PR, should be declared as a constant

Reply:
I have changed occurenses of `"yaml"`to `YAML` which is defined in `YamlVerify`